docs/roles/RoleNORManager.md

```diff
@@ -0,0 +1,43 @@
+## OBO Role: New Ontology Request (NOR) Manager
```

Comment:
```suggestion
---
layout: doc
permalink: /role/nor-manager
title: OBO Role: New Ontology Request (NOR) Manager
---
```

Permalink actually lets you specify what the address is, rather than using the sketchy looking path + name